docs/api_doc/creation/index.rst

```diff
@@ -3,15 +3,118 @@
+on the selected transformer. The transformers are to be used with numerical 
+variables, e.g., integer and float data types.
+
+Summary of feature-engine’s feature-creation transformers:
+
+   - CyclicalFeatures - Create two new features per original feature by applying 
+     the trigonometric operations sine and cosine to each original feature.
+   
+   - MathFeatures - Applies basic mathematical functions, which are used in 
+     ``pandas.agg()``, to subsets of features to create one or more new features.
+   
+   - RelativeFeatures - Utilizes basic mathematical functions between a group 
+     of variables and one or more reference features, appending the new features 
+     to the dataframe.
+
+In addition to the abovementioned transformer, feature-engine includes the 
+``SklearnTransformerWrapper()``, which applies a Scikit-learn transformer to a 
+selected set of features. ``SklearnTransformerWrapper()`` is similar to Scikit-learn’s 
+``ColumnTrasnformer()`` but differs in implementation.
+
+One useful application is to use ``SklearnTransformerWrapper()`` with Scikit-learn’s 
+``PolynomialFeatures()``. This is appropriate in feature engineering when an independent 
+variable does not have a linear or monotonic relationship with the target variable.
+
+A data scientist may perform feature creation when they believe that the interaction 
+between two variables can have a strong relationship with the target variable.
+
+feature-engine in Practice
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+Let’s use Kaggle’s wine quality dataset as an example. The dataset is comprised of 11 
+features, including ``alcohol``, ``pH``, and ``residual sugar``, and has ``quality`` as its 
+target variable.
+
+Through exploratory data analysis and our domain knowledge which includes real-world 
+experimentation, i.e., drinking various brands/types of wine, we believe that we can 
+create better features to train our algorithm by deriving the ratios of acidic-to-sweet 
+and alcohol-to-sweet.
+
+In our example, we import three Python packages - os, pandas, and feature-engine. We also 
+download the Kaggle dataset. We then use feature-engine’s ``RelativeFeatures()`` to perform 
+feature transformation.
+
+.. code-block:: python
+
+   import os
+   import pandas as pd
+   from feature_engine.creation import RelativeFeatures
+
+   # download and unzip Kaggle wine-quality raw data
+   os.environ["KAGGLE_USERNAME"] = "kaggle_username”
+   os.environ["KAGGLE_KEY"] = "kaggle_api_key"
+   !kaggle datasets download -d yasserh/wine-quality-dataset
+   !unzip -q wine-quality-dataset.zip
+   
+   # read raw data and extract relevant features
+   data = pd.read_csv("WineQT.csv")
+   df = data[["alcohol", "pH", "residual sugar"]].copy()
+   
+   # perform feature transformation
+   rf = RelativeFeatures(variables=["alcohol", "pH"], reference=["residual sugar"], func=["div"])
+   rf.fit(df)
+   df_tr = rf.transform(df)
```

Comment:
can we break this massive code block into smaller chunks and show the intermediate results in between?

For example load data and show the first rows, and then also show the result of the relative features transformation?